tests/transports-governance/test_customers_crud.py

```
@@ -0,0 +1,981 @@
+        # Update concurrently with different names
+        def update_customer(index):
+            update_data = {"name": f"Updated by thread {index}"}
+            response = governance_client.update_customer(customer_id, update_data)
+            return response, index
+
+        with ThreadPoolExecutor(max_workers=5) as executor:
+            futures = [executor.submit(update_customer, i) for i in range(5)]
+            results = [future.result() for future in futures]
+
+        # All updates should succeed (last one wins)
+        for response, index in results:
```

Ruff (0.12.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)